ts/screens/wallet/payment/TransactionSummaryScreen.tsx

```diff
@@ -471,8 +471,8 @@ const TransactionSummaryScreen = (): React.ReactElement => {
             () => startAssistanceRequest(error, messageId)
           )}
         </ContentWrapper>
-      </SafeAreaView>
-    </BaseScreenComponent>
+      </BaseScreenComponent>
```

Comment:
On Android the button is too close to the bottom. 

<img src="https://github.com/pagopa/io-app/assets/6160324/ee9aafbb-d8f9-443c-8b52-25bea853f9df" width="180" />


Maybe a `VSpacer` could be enough. What do you think?
```suggestion
          <VSpacer size={16} />
      </BaseScreenComponent>
```

Reply:
This issue is already addressed in another PR #5185

The actual PR aims to remove the heading spacing glitch.